projects/bitcoin-core/project.yaml

```diff
@@ -0,0 +1,20 @@
+homepage: "https://github.com/bitcoin/bitcoin"
+main_repo: 'https://github.com/bitcoin/bitcoin.git'
+language: c++
+primary_contact: "marco@chaincode.com"
+auto_ccs:
+  - "fanquake@gmail.com"
+  - "john@brink.dev"
+  - "jonas@chaincode.com"
+  - "laanwj@gmail.com"
+  - "pieter@chaincode.com"
+  - "thomas.j.bitcoin@protonmail.com"
```

Comment:
Noting that one needs to use a Google account to get full access to crash reports and fuzzer statistics [[source](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary)].